py/nightwatch/plots/camfiber.py

```diff
@@ -291,3 +296,156 @@ def plot_per_fibernum(cds, attribute, cameras, titles={},
         figs_list.append(fig)
         
     return figs_list
+
+def linregress_iter(x,y,sigclip=5):
+    sel = np.full(len(x), True)
+    for j in range(3):
+        linfit = scipy.stats.linregress(x[sel],y[sel])
+        residual = y-linfit.intercept-linfit.slope*x
+        sel = (residual<sigclip*np.std(residual[sel]))
+    return linfit
+
+def galactic_coord(header):
+    # Compute Galactic longitude and latitude
+    loc = SkyCoord(ra=header["SKYRA"]*u.degree, dec=header["SKYDEC"]*u.degree, frame='icrs')
+    return loc.galactic
+
+def radec_to_xyz(ra,dec):
+    ra = ra*np.pi/180.0
+    dec = dec*np.pi/180.0
+    return np.array([np.cos(ra)*np.cos(dec), np.sin(ra)*np.cos(dec), np.sin(dec)])
+
+def moon_elevation(header):
+    # Compute the elevation of the moon
+    lst = header["TCSST"].split(":")
+    zenith = radec_to_xyz(15*(float(lst[0])+float(lst[1])/60.0+float(lst[2])/3600.0), float(header["OBS-LAT"]))
+    moon = radec_to_xyz(header["MOONRA"], header["MOONDEC"])
+    return np.arcsin(np.dot(zenith,moon))*180.0/np.pi
+
+def sun_elevation(header):
+    # Compute the elevation of the sun and phase of moon
+    lst = header["TCSST"].split(":")
+    zenith = radec_to_xyz(15*(float(lst[0])+float(lst[1])/60.0+float(lst[2])/3600.0), float(header["OBS-LAT"]))
+    try: sun = radec_to_xyz(header["SUNRA"], header["SUNDEC"])
+    except:
+        # These keywords were only added in mid-February 2021; return False if not available.
+        return False, False
+    moon = radec_to_xyz(header["MOONRA"], header["MOONDEC"])
+    phase = (1.0-np.arccos(np.dot(moon,sun))/np.pi)   # 0 = moon, sun together; 1 = opposed.
+    return np.arcsin(np.dot(zenith,sun))*180.0/np.pi, phase
+
+def plot_fractionalresidual(fiber, header, expos, position=False, plot_height=500, plot_width=500):
```

Comment:
Please add docstrings to all functions defining their inputs and outputs so that someone would know how to call the function without having to read the code.  e.g. what is `fiber`, and `expos`, and what would `position=True` do?